src/EditorFeatures/CSharpTest/ExtractInterface/ExtractInterfaceTests.cs

```diff
@@ -1081,11 +1081,11 @@ public class $$C
 
                 var handler = new ExtractInterfaceCommandHandler();
                 var delegatedToNext = false;
-                CommandState nextHandler()
+                Func<CommandState2> nextHandler = () =>
```

Comment:
why change this from a local function to a lambda?

Reply:
mismerge will fix